### PR TITLE
[IMP] Add fixture in order to launch an instance of odoo http server

### PR DESCRIPTION
**Pull request by rousseldenis**
@guewen @simahawk

### CODE REVIEW

**Review by guewen — APPROVED** [review]
That's so cool!
Thanks

File: pytest_odoo.py [hunk 1]
```diff
@@ -79,6 +78,13 @@ def pytest_cmdline_main(config):
         yield
 
 
+@pytest.fixture(scope="module", autouse=True)
```
Comment (sbidoul) [inline pytest_odoo.py:81]: Any reason to make it module level than session level?

Comment (rousseldenis) [reply pytest_odoo.py:81]: Yes, it seems that server is killed when entering testing functions using session.

**Review by simahawk — APPROVED** [review]
LG and tested! Thanks! :heart:

**Comment by simahawk** [comment]
pypi publishing is broken, checking why

**Comment by simahawk** [comment]
I pushed manually :man_shrugging: 

https://pypi.org/project/pytest-odoo/1.0.0/

**Pull request merged**